pkg/webui/components/footer/index.js

```diff
@@ -50,9 +50,16 @@ const Footer = function({ className, links, supportLink, isOnline }) {
         ))}
         <OfflineStatus isOnline={isOnline} showOfflineOnly showWarnings />
         <span className={style.version}>v{process.env.VERSION}</span>
+        <Link.Anchor
+          className={style.documentation}
+          secondary
+          href="https://www.thethingsindustries.com/docs"
```

Comment:
This should be taken from the configuration (`APP_CONFIG.documentation_base_url`).
See https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/pkg/webui/lib/env.js#L30